pkg/blackduck/installer/crdinstaller.go

```diff
@@ -203,7 +203,7 @@ func (c *CRDInstaller) Deploy() error {
 
 // PostDeploy will call after deploying the CRD
 func (c *CRDInstaller) PostDeploy() {
-	protoform.SetupHTTPServer(c.kubeClient, c.hubClient, c.config.Namespace)
+	//protoform.SetupHTTPServer(c.kubeClient, c.hubClient, c.config.Namespace)
```

Comment:
It's need for prometheus metrics right.. May be we can move this while installing the operator instead of having it in blackduck.. I have removed the metrics that logs every 5 secs in the PR which is already merged with the master.

Reply:
Good idea. I will move this outside of the pkg/blackduck.